Introduce how to use unofficial ros_gz combinations for Garden/Humble (#362)
gazebosim/docs: ros_installation.md

</div>

To select a different release of Gazebo than the one officially supported by
your ROS distribution you must either **use non ROS official Gazebo binary packages** or
**manually compile**
[`ros_gz`](https://github.com/gazebosim/ros_gz) from source (this rule applies for
every ROS package that uses a Gazebo library):

>   * **Use non ROS official ros_gz binary packages:** *packages.osrfoundation.org* provides

Reviewer: Or more correctly "Unofficial ROS" versus "non ROS official."

    binary packages for `ros_gz` for some combinations of Gazebo and ROS that are different

>     from the existing in ROS. In some circumstances these packages can be outdated or

Reviewer 2: Is "existing" saying the officially recommended / supported combination? Using the same wording would be less confusing
"these" is saying the non-official ones, right? It is easily misinterpreted because it follows right after the last words in the previous sentence that's saying the official ones.
```suggestion
    from the officially supported combination. In some circumstances the non-official packages can be outdated or
```

    even buggy, the use of them is reserved for advanced users.

  * **Manually compile ros_gz:** some combinations of Gazebo and ROS can be prepared to be

>     built together but don't have official or unofficial binary packages.

Reviewer 2: Being more explicit / clearer
```suggestion
    built together but have no binary packages, neither official nor unofficial.
```

Both approaches may also require that you modify your ROS or Gazebo source code to support this compilation.

### Gazebo Garden (Not Recommended)

> Gazebo Garden can be used with ROS 2 Humble and non official ROS binary packages hosted

Reviewer 2: To be consistent with above
"non ROS official"
Or maybe use hyphens to make it more clear
"non-ROS-official" everywhere?
Actually I see that a little below, it uses "unofficial." That's a shorter word, and I prefer it. Maybe just replace "non ROS official" and "non official ROS" with "unofficial" everywhere.
```suggestion
Gazebo Garden can be used with ROS 2 Humble and non ROS official binary packages hosted
```

Reviewer: I concur, but "unofficial ROS" is the official replacement for "non-official ROS." 🫠

in packages.osrfoundation.org. These packages conflict with `ros-humble-ros-gz*`

> packages (based on Gazebo Fortress).

Reviewer 2: Being more explicit
```suggestion
packages (Humble officially supports Gazebo Fortress).
```

To install the binary Gazebo Garden/ROS 2 Humble packages:

> 

Reviewer 2: ```suggestion
```

 * Folow [these instruction to install gz-garden](https://gazebosim.org/docs/garden/install_ubuntu#binary-installation-on-ubuntu)

>    from `osrfoundation.org` repository

Reviewer: Instead of saying  `from the osrfoundation.org repository..` I would just point directly to where the binary packages live.  OSRFoundation.org just redirects to openrobotics.org so it is really unclear to me what this is trying to point to.

>  * Install the unofficial binary packages from apt:

Reviewer 2: This package is for `ros_gz`?
```suggestion
 * Install `ros_gz` from the unofficial binary packages from apt:
```
